text/0002-rfc-process.md

```diff
@@ -83,7 +83,7 @@ Whomever merges the RFC should do the following:
 * Create a corresponding issue on [Rust repo](https://github.com/rust-lang/rust)
 * Fill in the remaining metadata in the RFC header, including links for
   the original pull request(s) and the newly created Rust issue.
-* Add an entry in the [Active RFC List] of the root `README.md`.
+* Add an entry in the [Active RFC List](https://rfcbot.rs/) of the root `README.md`.
```

Comment:
changing the link won't match the subsequent text though. 

as the entire point is outdated, but we don't update the content of an outdated RFC, imo we shouldn't touch this link at least.

Reply:
I see, I understand